ql/src/test/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/TestHiveMetaStoreAuthorizer.java

```diff
@@ -411,4 +418,93 @@ public void testU_DropDataConnector_authorizedUser() {
       fail("testU_DropDataConnector_authorizedUser() failed with " + e);
     }
   }
+
+  /**
+   * Fake tables to test.
+   */
+  private static class TestTable {
+    private final String dbName;
+    private final String tableName;
+
+    private TestTable(String dbName, String tableName) {
+      this.dbName = dbName;
+      this.tableName = tableName;
+    }
+
+    @Override
+    public String toString() {
+      return dbName + "." + tableName;
+    }
+
+    public String getDbName() {
+      return dbName;
+    }
+
+    public String getTableName() {
+      return tableName;
+    }
+  }
+
+
+  private static List<TestTable> createTables(int dbs, int tbls) {
+    List<TestTable> tables  = new ArrayList<>(dbs * tbls);
+    for(int d = 0; d < dbs; ++d) {
+      for(int t = 0; t < tbls; ++t) {
+        String dbname = String.format("db%05x", d);
+        String tblName = String.format("tbl%05x", t);
+        tables.add(new TestTable(dbname, tblName));
+      }
+    }
+    return tables;
+  }
+
+  private static final int NUM_DB = 2;
+  private static final int NUM_TBL = 50;
+  @Ignore
+  public void testIndexedTableLookup0() {
```

Comment:
Not sure of the purpose of this test, could you clarify or remove?

Reply:
Removed.